etcdserver/raft.go

```diff
@@ -85,6 +85,15 @@ type apply struct {
 	raftDone <-chan struct{} // rx {} after raft has persisted messages
 }
 
+// activeRaft represents an activated raft instance via raft.{Res,S}tartNode
+type activeRaft struct {
```

Comment:
I feel we need a better name for this struct. "activeRaft, raftNode, raft.Node": we have too many rafts in etcdserver.